src/Components/Common/BloodPressureFormField.tsx

```diff
@@ -112,4 +85,14 @@ export const BloodPressureValidator: FieldValidator<BloodPressure> = (bp) => {
   if (bp.systolic == null) {
     return "Systolic is missing. Either specify both or clear both.";
   }
+  if (bp.systolic > 250) {
+    return "Systolic value cannot exceed 250.";
+  }
+
+  if (bp.diastolic > 250) {
+    return "Diastolic value cannot exceed 250.";
+  }
+  if (bp.systolic < bp.diastolic) {
+    return "Blood Pressure - Systolic must be greater than diastolic";
+  }
```

Comment:
Keep error messages in i18n en locale file

Reply:
Hey @rithviknishad, should I consolidate error messages related to specific fields or display each individually?
<img width="943" alt="Screenshot 2024-10-14 at 2 40 29 PM" src="https://github.com/user-attachments/assets/d03c34fd-26e5-43da-9af5-a0993a5d5951">

Reply:
You can consolidate and make it generic as same validations apply for both.

Reply:
```
"blood_pressure_error": {
  "missing": "Blood pressure values are missing. Either specify both or clear both.",
  "exceed": "Blood pressure value cannot exceed 250.",
  "systolic_less_than_diastolic": "Blood Pressure - Systolic must be greater than diastolic."
}
```
Like this ?

Reply:
```
"blood_pressure_error": {
  "missing": "Field is required. Either specify both or clear both.",
  "exceed": "Value cannot exceed 250 mmHg.",
  "systolic_less_than_diastolic": "Systolic must be greater than diastolic."
}
```